pyomo/gdp/plugins/bigm.py

```diff
@@ -251,8 +251,8 @@ def _transform_blockData(self, obj, bigM):
                 descent_order=TraversalStrategy.PostfixDFS):
             self._transform_disjunction(disjunction, bigM)
 
-    def _get_xor_constraint(self, disjunction):
-        # Put the disjunction constraint on its parent block and
+    def _get_xor_constraint(self, disjunction, transBlock):
+        # Put the disjunction constraint on the transformation block and
         # determine whether it is an OR or XOR constraint.
```

Comment:
The name of the function and its description seem to diverge slightly. `get` operations usually don't modify state.